pkg/tests/integration/test_multi_minion.py

```diff
@@ -0,0 +1,127 @@
+import os
+import pathlib
+import subprocess
+
+import psutil
+import pytest
+
+pytestmark = [
+    pytest.mark.skip_unless_on_windows,
+]
+
+
+@pytest.fixture
+def mm_script(install_salt):
+    yield install_salt.ssm_bin.parent / "multi-minion.ps1"
+
+
+@pytest.fixture(scope="function")
+def mm_conf(mm_script):
+    yield pathlib.Path(os.getenv("LocalAppData"), "Salt Project", "Salt", "conf")
+    subprocess.run(
+        ["powershell", str(mm_script).replace(" ", "' '"), "-d"],
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        check=False,
+        universal_newlines=True,
+    )
+
+
+def test_script_present(mm_script):
+    """
+    Ensure the multi-minion.ps1 file is present in the root of the installation
+    """
+    assert mm_script.exists()
+
+
+def test_install(mm_script, mm_conf):
+    """
+    Install a 2nd minion with default settings. Should create a minion config
```

Comment:
**Suggestion (style, non-blocking):** I recommend changing to "second" for clarity. https://developers.google.com/style/numbers

Also, I'm not sure about saying "should create." That sounds a little ambiguous. Maybe "Creates a minion config."